src/experiments/logins/api.js

```diff
@@ -49,6 +49,20 @@ this.logins = class extends ExtensionAPI {
           //   - fooLogin = login as vanilla JS object
           //   - fooLoginInfo = login as nsILoginInfo
           //   - fooBag = (subset of) login fields as nsIPropertyBag
+          getLoginSavingEnabled(aHost) {
+            try {
+              return Services.logins.getLoginSavingEnabled(aHost);
+            } catch (ex) {
+              throw new ExtensionError(ex);
+            }
+          },
+          setLoginSavingEnabled(aHost, isEnabled) {
```

Comment:
How about `origin` and `shouldSaveLogins` for the arguments?

Reply:
Yeah, same as above, I was trying to follow the underlying API rather than invent new names